frontend/src/pages/search/index.tsx

```diff
@@ -350,31 +358,20 @@ const Search = () => {
                   </div>
                 </div>
                 <div className="md:w-3/4">
-                  <div className="md:py-8 md:pl-8">
-                    <p className="sm:hidden mt-4 mb-2">{placeholder}</p>
-                    <SearchForm
-                      placeholder={placeholder}
-                      handleSearchInput={handleSearchInput}
-                      input={searchCriteria.query_string}
-                    />
-                    <div className="flex justify-end mt-3">
-                      <ExactMatch
-                        checked={searchCriteria.exact_match}
-                        id="exact-match"
-                        handleSearchChange={handleSearchChange}
-                      />
-                      <div className="ml-1 -mt-1 text-sm">
-                        <Tooltip id="exact_match" tooltip={exactMatchTooltip} />
-                      </div>
-                    </div>
-                  </div>
                   <div className="mt-4 relative z-10">
                     <TabbedNav
                       activeIndex={categoryIndex}
                       items={documentCategories}
                       handleTabClick={handleDocumentCategoryClick}
                     />
-                    <div className="mt-4 md:absolute right-0 top-0 md:-mt-2 flex z-10">
+                    <div className="mt-4 md:absolute right-0 top-0 md:-mt-8 lg:-mt-4 flex z-10 items-center">
```

Comment:
Sorry I know not strictly part of this ticket, but the sort select is appearing above the menu opening

Reply:
@PatFawbertMills not sure what you mean here, can you screenshot it?

Reply:
![Screenshot 2022-06-21 at 09 18 37](https://user-images.githubusercontent.com/10725753/174751850-4638e3a4-22c2-4414-b77e-6692fe274612.png)

Reply:
@PatFawbertMills This should be fixed now, please double check.